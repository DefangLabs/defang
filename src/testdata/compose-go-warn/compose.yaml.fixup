{
  "echo": {
    "command": null,
    "deploy": {
      "replicas": 1,
      "resources": {},
      "placement": {}
    },
    "entrypoint": null,
    "image": "ealen/echo-server",
    "networks": {
      "default": null
    },
    "ports": [
      {
        "mode": "ingress",
        "target": 80,
        "protocol": "tcp",
        "app_protocol": "http"
      }
    ]
  }
}