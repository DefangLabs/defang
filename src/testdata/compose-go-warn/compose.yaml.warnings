 ! "yes" for boolean is not supported by YAML 1.2, please use `true`
 ! can only have more than one replica for PRO tier and PRODUCTION deployment mode. Will set replicas to 1 for service(s) [echo]
 ! service "echo": ingress port without healthcheck defaults to GET / HTTP/1.1
 ! service "echo": missing memory reservation; using provider-specific defaults. Specify deploy.resources.reservations.memory to avoid out-of-memory errors