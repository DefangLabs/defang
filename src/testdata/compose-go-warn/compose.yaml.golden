name: compose-go-warning
services:
  echo:
    image: ealen/echo-server
    networks:
      default: null
    ports:
      - mode: ingress
        target: 80
        protocol: tcp
networks:
  default:
    name: compose-go-warning_default
secrets:
  dummy:
    name: dummy
    external: true
